RevenueCatUI/PaywallView.swift

```diff
@@ -227,6 +236,8 @@ struct LoadedOfferingPaywallView: View {
             .preference(key: PurchasedCustomerInfoPreferenceKey.self,
                         value: self.purchaseHandler.purchasedCustomerInfo)
             .disabled(self.purchaseHandler.actionInProgress)
+            .onAppear { self.purchaseHandler.trackPaywallView(self.eventData) }
+            .onDisappear { self.purchaseHandler.trackPaywallClose() }
```

Comment:
does onDisappear get called when the app gets backgrounded? And when the iOS payment window pops up?

Reply:
wondering whether we need to [do something like this](https://lyons.app/2020/08/31/how-to-only-run-ondisappear-when-a-view-is-unloaded-in-swiftui/#:~:text=How%20to%20only%20run%20onDisappear%20when%20a%20view,code%20when%20the%20view%20is%20dismissed%20or%20unloaded.)

Reply:
Good point, let me test this + that approach.

Reply:
The answer to both is no :) so the simple approach here works perfectly without extra hacks :)